content/en/flux/installation/bootstrap/bitbucket.md

```diff
@@ -5,26 +5,40 @@ description: "How to bootstrap Flux with Bitbucket Server and Data Center"
 weight: 40
 ---
 
-### Bitbucket Server and Data Center
-
-The `bootstrap bitbucket-server` command creates a Bitbucket Server repository if one doesn't exist and
-commits the Flux components manifests to the specified branch. Then it
-configures the target cluster to synchronize with that repository by
-setting up an SSH deploy key or by using token-based authentication.
+The [flux bootstrap bitbucket-server](/flux/cmd/flux_bootstrap_bitbucket-server/) command deploys the Flux controllers	
+on a Kubernetes cluster and configures the controllers to sync the cluster state from a Bitbucket project.	
+Besides installing the controllers, the bootstrap command pushes the Flux manifests to the Bitbucket project	
+and configures Flux to update itself from Git.	
+
+After running the bootstrap command, any operation on the cluster (including Flux upgrades)	
+can be done via Git push, without the need to connect to the Kubernetes cluster.
+
+{{% alert color="danger" title="Required permissions" %}}
+To bootstrap Flux, the person running the command must have **cluster admin rights** for the target Kubernetes cluster.
+It is also required that the person running the command to be the **owner** of the Bitbucket project,
+or to have admin rights of a Bitbucket group.
```

Comment:
Does Bitbucket have the concept of groups?

Reply:
Yes. 

Source 1: https://support.atlassian.com/bitbucket-cloud/docs/organize-groups-for-your-repositories/
Source 2: The original bitbucket section has this line that states "_When you specify a list of groups, those teams will be granted write access to the repository._" https://fluxcd.io/flux/installation/#bitbucket-server-and-data-center